src/lib/pmd/pmd-rule-cataloger/src/main/java/sfdc/isv/swat/CatalogCategory.java

```diff
@@ -0,0 +1,38 @@
+package sfdc.isv.swat;
```

Comment:
Should put these in a sub package.  Actually, everything here may need to be more scanner-specific.  We also probably do not need sfdc.isv.swat, vs. just sfdc.isv_swat or sfdc.swat

And then use ...swat.scanner

And then for the JSON stuff, use ...scanner.catalog or ....scanner.json etc.